lambda/MAS/Services/S3Service.cs

```diff
@@ -25,13 +23,13 @@ public S3Service(IAmazonS3 amazonS3, ILogger<S3Service> logger)
             _logger = logger;
         }
 
-        public async Task<PutObjectResponse> WriteToS3(Item item)
+        public async Task<PutObjectResponse> WriteToS3(Item item, string body)
```

Comment:
If we're only using item for the path of the item, then I don't think this makes sense for the service to have a whole item, it might as well just take a path and a body